WordPress/UITestsFoundation/Screens/StatsScreen.swift

```diff
@@ -16,6 +23,38 @@ public class StatsScreen: ScreenObject {
         )
     }
 
+    public func verifyStatsAreLoaded(_ stats: [String]) -> Bool {
```

Comment:
Just a naming nitpick (applicable to all four newly created methods): articles in methods names are generally not necessary. It's [not against the rules](https://github.com/wordpress-mobile/WordPress-iOS/pull/17536#discussion_r755699987) at the same time. 🙂 

```suggestion
    public func verifyStatsLoaded(_ stats: [String]) -> Bool {
```

(I must admit, I just checked my Simplenote tests, and they also contain articles in some cases 😅).

Reply:
Oops, I did it again. ®️ 
Would it be the same for assertion methods like `assertStatsAreLoaded()`?  🤔

Reply:
> Would it be the same for assertion methods like assertStatsAreLoaded()?

I tend to say yes, because from a code style guide POV (which doesn't really cover this point), it should be the same for all cases.